<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1182</width>
    <height>904</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string>实时演示</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*#left_widget{
	background-color:#c1c1c1;
}*/

QWidget #view_widget{
	background-color:#cdcdcd;
}
QLabel[isFullLabel=true]{/*满载提示统一设置*/
	background-color:rgb(55,173,107);
	color:white;
	font:bold;
	font-size:25px;
}
QLabel[isFulled=true]{/*满载变红*/
	background-color:#c93756;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="left_widget" native="true">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="9">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_7" stretch="13,3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QScrollArea" name="image_scroll">
            <property name="toolTip">
             <string>预备投放顺序提示</string>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>333</width>
               <height>725</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="image_scroll_label">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <italic>false</italic>
                  <underline>false</underline>
                 </font>
                </property>
                <property name="text">
                 <string>投放顺序提示</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="image_scroll_spacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>663</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,9">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>满载提示：绿-空，红-满</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_2" native="true">
               <property name="toolTip">
                <string>满载提示区域</string>
               </property>
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="fullLoad_harmful">
                  <property name="text">
                   <string>有害垃圾</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="isFullLabel" stdset="0">
                   <bool>true</bool>
                  </property>
                  <property name="isFulled" stdset="0">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="fullLoad_general">
                  <property name="text">
                   <string>其他垃圾</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="isFullLabel" stdset="0">
                   <bool>true</bool>
                  </property>
                  <property name="isFulled" stdset="0">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="fullLoad_recycle">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>可回收垃圾</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="isFullLabel" stdset="0">
                   <bool>true</bool>
                  </property>
                  <property name="isFulled" stdset="0">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="fullLoad_kitchen">
                  <property name="text">
                   <string>厨余垃圾</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="isFullLabel" stdset="0">
                   <bool>true</bool>
                  </property>
                  <property name="isFulled" stdset="0">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="right_widget" native="true">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>50</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QWidget" name="view_widget" native="true">
          <property name="toolTip">
           <string>实时演示</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="work_screen">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <italic>true</italic>
               <underline>true</underline>
              </font>
             </property>
             <property name="text">
              <string>视频幕布</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="terminal_widget" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>200</height>
           </size>
          </property>
          <property name="toolTip">
           <string>投放信息提示区域</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QScrollArea" name="throw_scroll">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_2">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>812</width>
                <height>172</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QWidget" name="widget_3" native="true">
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>完</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>成</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>顺</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>序</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>730</width>
                   <height>17</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
